listenbrainz/webserver/login/__init__.py

```diff
@@ -32,6 +32,16 @@ def from_dbrow(cls, user):
             login_id=user['login_id'],
         )
 
+    def to_dict(self):
```

Comment:
Ideally, I'd have converted dicts to object but this way was less changes so this way for now.